src/FusedChannelData.ts

```diff
@@ -36,7 +37,7 @@ export default class FusedChannelData {
   private channelsDataToFuse: Channel[];
 
   private fuseGeometry: PlaneGeometry;
-  private fuseMaterial: ShaderMaterial;
+  private fuseMaterial: ShaderMaterial[];
```

Comment:
we now need separate shaders here to handle different possible channel data types: uint, int or float

Reply:
Since there are only 3, I just made them explicit.  If the number expands, I'll look into a named mappable refactor or something.